services/QuillLMS/spec/queries/snapshots/active_students_query_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Snapshots
+  describe 'ActiveStudentsQuery' do
+    context 'external_api', :external_api do
+      before do
```

Comment:
I would add 
```
  config.around(:each, :external_api) do |example|
    VCR.configure { |c| c.allow_http_connections_when_no_cassette = true }
    example.run
    VCR.configure { |c| c.allow_http_connections_when_no_cassette = true }
  end
```
to spec_helper similar to the `caching` tag [configuration](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/spec/rails_helper.rb#L92)

Then you can get rid of all the before after VCR in all of these tests in addition to the ones that Peter wrote